src/overlays/actors/ovl_En_Hy/z_en_hy.h

```diff
@@ -6,9 +6,34 @@
 
 struct EnHy;
 
+typedef void (*EnHyActionFunc)(struct EnHy*, GlobalContext*);
+
 typedef struct EnHy {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1E8];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ EnHyActionFunc actionFunc;
+    /* 0x0190 */ char unk_190;
```

Comment:
I believe the offset here is incorrect.
```suggestion
    /* 0x0194 */ char unk_194;
```